src/main/resources/com/sonyericsson/jenkins/plugins/bfa/db/MongoDBKnowledgeBase/config.jelly

```diff
@@ -41,11 +41,14 @@
     <f:entry title="${%Use TLS connection}">
         <f:checkbox field="tls" />
     </f:entry>
+    <f:entry title="${%Disable RetryWrites}">
```

Comment:
This makes it even more confusing.

Reply:
Needs a help file and better naming of the field.

Reply:
Simplified the change